spec/components/custom_wizard/mapper_spec.rb

```diff
@@ -250,4 +274,75 @@
       user: user1
     ).perform).to eq(false)
   end
+
+  context "Output Templating" do
```

Comment:
@fzngagan We need a test for the case when ``template: false``.

Reply:
I've added a test for this